app/models/announcement.rb

```diff
@@ -1,2 +1,21 @@
+# frozen_string_literal: true
+
+require 'active_support/all'
+
 class Announcement < ApplicationRecord
+  def self.populate_from_airtable
+    client = Airtable::Client.new(ENV['airtable_api_key'])
+    table = client.table(Setting.last.airtable_base_id, Setting.last.announcements_table_id)
+    records = table.records
+    Announcement.delete_all
+    records.each do |record|
+      recorded_author = if record[:submitteremail].length.zero?
+                           'BSFC'
```

Comment:
magic string?

Reply:
This is intentional.  The author should be 'BSFC' in all cases unless a certain airtable field is filled out

Reply:
i get that, i just mean this should be a constant declared somewhere at the top of an appropriate file rather than a string embedded deep in a method somewhere.  it's impossible to document what your comment says otherwise.  i'll accept for the merge but please do a separate PR afterward to fix